testsuite/tests/typing-private/private.compilers.reference

```diff
@@ -80,48 +79,48 @@ Error: Signature mismatch:
          type t = int
        is not included in
          type t = private Foobar.t
-#               module M : sig type t = private T of int val mk : int -> t end
-#               module M1 : sig type t = M.t val mk : int -> t end
-#             module M2 : sig type t = M.t val mk : int -> t end
-#         module M3 : sig type t = M.t val mk : int -> t end
-#         Characters 21-44:
+module M : sig type t = private T of int val mk : int -> t end
+module M1 : sig type t = M.t val mk : int -> t end
+module M2 : sig type t = M.t val mk : int -> t end
+module M3 : sig type t = M.t val mk : int -> t end
+Characters 21-44:
       type t = M.t = T of int
       ^^^^^^^^^^^^^^^^^^^^^^^
 Error: This variant or record definition does not match that of type M.t
        A private type would be revealed.
-#           module M5 : sig type t = M.t = private T of int val mk : int -> t end
-#         module M6 : sig type t = private T of int val mk : int -> t end
-#                   module M' :
+module M5 : sig type t = M.t = private T of int val mk : int -> t end
+module M6 : sig type t = private T of int val mk : int -> t end
+module M' :
   sig type t_priv = private T of int type t = t_priv val mk : int -> t end
-#         module M3' : sig type t = M'.t val mk : int -> t end
-#     module M : sig type 'a t = private T of 'a end
-#     module M1 : sig type 'a t = 'a M.t = private T of 'a end
-#       module Test : sig type t = private A end
+module M3' : sig type t = M'.t val mk : int -> t end
+module M : sig type 'a t = private T of 'a end
+module M1 : sig type 'a t = 'a M.t = private T of 'a end
+module Test : sig type t = private A end
 module Test2 : sig type t = Test.t = private A end
-# val f : Test.t -> Test2.t = <fun>
-# val f : Test2.t -> unit = <fun>
-# Characters 8-15:
+val f : Test.t -> Test2.t = <fun>
+val f : Test2.t -> unit = <fun>
+Characters 8-15:
   let a = Test2.A;; (* fail *)
           ^^^^^^^
 Error: Cannot create values of the private type Test2.t
-# *   Characters 148-171:
+Characters 148-171:
   module Test2 : module type of Test with type t = private Test.t = Test;;
                                           ^^^^^^^^^^^^^^^^^^^^^^^
 Warning 3: deprecated: spurious use of private
 module Test2 : sig type t = Test.t = private A end
-#     type t = private < x : int; .. >
-# type t = private < x : int; .. >
-# type t = private < x : int >
-# type t = private < x : int >
-# Characters -1--1:
+type t = private < x : int; .. >
+type t = private < x : int; .. >
+type t = private < x : int >
+type t = private < x : int >
+Characters -1--1:
```

Comment:
This is independent from the current PR but it seems to be a bug of the error printing code: the range `(-1)-(-1)` should not be displayed to the user, as it is nonsensical.